src/BinSkim.Driver/BinSkim.Driver.csproj

```diff
@@ -68,6 +68,7 @@
   <ItemGroup>
     <!-- May need to be changed to x86 on a 32 bit machine. -->
     <MsDiaLibs Include="..\..\refs\x64\*" Condition="'$(TargetFramework)'=='$(NetCoreVersion)' And '$(RuntimeIdentifier)'==''" />
```

Comment:
>$(TargetFramework) [](http://example.com/codeflow?start=54&length=18)

This macro doesn't exist anymore.
Do not need the additional <MsDiaLibs> since both `NetCoreVersion` and `NetSixVersion` need this tag. Just remove the TargetFramework condition.